backend/user/src/server.ts

```diff
@@ -12,6 +13,7 @@ app.use(pino());
 app.use(json());
 app.use(helmet());
 app.use('/auth', authRoutes);
+app.use('/register', registerRoutes);
```

Comment:
@sembcorpp might wanna discuss with @TabrizPlv if this is the routing to go with, or to merge with auth routes